cron.d/LINK_CHECKER

```diff
@@ -0,0 +1,4 @@
+MAILTO=natalia.atkins@utas.edu.au
+
+5 8 * * 7 projectofficer bin/geonetwork_broken_link_checker.py --geonetwork-url  http://catalogue-systest.aodn.org.au/geonetwork --email-to natalia.atkins@utas.edu.au
```

Comment:
by adding MAILTO, she will receive 2 emails. one from the cron, plus one from the script since there is already a --mail-to . Both outputs will be different